src/ts/containers/Home.tsx

```diff
@@ -1,13 +1,15 @@
+import { browser } from 'webextension-polyfill-ts'
```

Comment:
maybe we can wrap this in a service so that it won't need to be imported in all components. the service will be used to get message by key

Reply:
yes, good idea